BioCro/DARPA/biocro_regional_darpa_files/subset_weather.Rmd

```diff
@@ -0,0 +1,148 @@
+
+### 2. Determine desired dimensions
+
+Use the following plotting functionality in R to figure out which latitudes and longitudes to clip the weather data to. 
+
+```{r message=FALSE, warning=FALSE}
+library(ncdf4)
+library(dplyr)
+library(maps)
+library(ggplot2)
+```
+
+The first example shows the four corners of a long latitudinal slice that includes area in both China and Russia. Change out the `lat` and `lon` values in the data frame and the filtered region in the `background_map` as desired. 
+
+```{r}
+chiruss_corners = expand.grid(lat = c(44.9, 55.1), 
+                             lon = c(123.9, 127.1))
+background_map <- map_data("world") %>% 
+  filter(region == "China" | region == "Russia")
+ggplot() +
+  geom_polygon(data = background_map, aes(x = long, y = lat, group = group), fill = "white", color = "black") +
+  geom_point(data = chiruss_corners, aes(x = lon, y = lat), color = "red")
+```
+
+This plot shows the limits of the NARR `all.nc` file. 
+
+```{r}
+narr_all_corners = expand.grid(lat = c(10.125, 83.875), 
+                             lon = c(-169.875, -50.125))
+background_map <- map_data("world")
+ggplot() +
+  geom_polygon(data = background_map, aes(x = long, y = lat, group = group), fill = "white", color = "black") +
+  geom_point(data = narr_all_corners, aes(x = lon, y = lat), color = "red")
+```
+
+### 3. Use nco to subset file
+
+Navigate in the HPC web interface to `/xdisk/username/`. The .nc files should be there now. You can see the subsetting command by looking at the first line under history after running this: 
+
+```{shell}
+ncdump -h champaign.nc
+```
+
+The command is below. This is what was used to generate `champaign.nc` from `all.nc`, and will be the basis for creating new subsets. The `ncks` arguments -O is for overwrite, and -d is for dimensions for specified variables. 
+
+```{shell}
+ncks -O -d longitude,-88.9,-87.5 -d latitude,39.8,40.5 all.nc champaign.nc
+```
+
+The following are modified commands for different weather subsets, including new file names. 
+
+Eastern Illinois: 
+```{shell}
+ncks -O -d longitude,-88.9,-87.5 -d latitude,38.8,41.5 all.nc eaill.nc
+```
+
+USA/Canada: 
+```{shell}
+ncks -O -d longitude,-108.5,-105.5 -d latitude,45.5,55.5 all.nc usca.nc
+```
+
+Russia/China (*this will only work with the CRUNCEP all.nc*):
+
+```{shell}
+ncks -O -d longitude,123.9,127.1 -d latitude,44.9,55.1 all.nc chiruss.nc
+```
+
+### 4. Copy to local machine using Globus
+
+Once the desired subset has been generated, you will need to transfer the file from the HPC collection `arizona#sdmz-dtn` on Globus to your own endpoint. This will require downloading Globus Connect Personal and following [these instructions](https://www.globus.org/globus-connect-personal) to create an endpoint on your machine. 
```

Comment:
The link isn't for instructions. Perhaps move the link to 'Globus Connect Personal' and change to '... and follow the installation and configuration instructions to create an ...'